docs/syntax_and_semantics/compile_time_flags.md

```diff
@@ -145,9 +145,9 @@ Crystal program.
 | `without_openssl` | Build without OpenSSL support
 | `without_zlib` | Build without Zlib support
 
-### Compiler features
+### Language features
 
-These flags enable or disable compiler features when building a Crystal program.
```

Comment:
dll's options do not belong here now. maybe move them to ###Compiler options below?

Reply:
`preview_dll` is already removed in #791. I merge `release/1.14` into master in order to reflect this.

Reply:
yeah, I noted that right after, but this is still valid for `preview_win32_delay_load`

Reply:
That was removed in #792

Reply:
I've merged `release/1.14` into `master` and `master` back into this PR branch, so the flags are now up to date.

Reply:
ah, sorry that I missed that